scopes/compilation/compiler/types.ts

```diff
@@ -1,11 +1,17 @@
 import { BuildContext, BuildTask, BuiltTaskResult, TaskResultsList } from '@teambit/builder';
+import ConsumerComponent from '@teambit/legacy/dist/consumer/component';
```

Comment:
having something from teambit/legacy here is problematic. let's discuss it.

Reply:
we can probably use `Consumer` instead, let me check